source/agora/node/Validator.d

```diff
@@ -421,10 +421,6 @@ public class Validator : FullNode, API
     {
         return new Clock((out long time_offset)
             {
-                // not enrolled - no need to synchronize clocks
```

Comment:
I think even if we remove this, the clock will not be synchronized, if the validator is not enrolled:
https://github.com/bosagora/agora/blob/v0.x.x/source/agora/network/Manager.d#L559-L562

for a non-enrolled validator the quorum set will be empty

Reply:
Yes so even better we remove the check here.